### DISCUSSION TITLE
Unable to launch FINN docker container

### DISCUSSION THREAD

**Discussion by Madhurima8** [Xilinx/finn#932] [Q&A] [answered]
I am facing an issue while installing FINN with docker. when I ran this command to verify the installation – ./run-docker.sh quicktest, I got the below error -
![thumbnail_image001](https://github.com/Xilinx/finn/assets/11993190/6bbf63bc-be20-4c55-be57-b2bdffba276c)

Is there a way to resolve this?

**Comment by fpjentzsch** [comment]
Hi, does this error also occur if you use the FINN dev branch? I see some warnings related to name resolution. Maybe you have an internet or DNS issue that prevents installation of the dependencies?

**Comment by sainadh812** [comment]
how to clone FINN dev branch ??

**Reply by auphelia** [reply, marked as answer]
For example:
`git clone https://github.com/Xilinx/finn.git`
`git checkout dev`
`bash run-docker.sh` to build the Docker environment.
